### PR TITLE
MOTECH-2117 Removing Gerrit from the repo doc

### PR DESCRIPTION
**Pull request by craigappl**

### CODE REVIEW

File: docs/source/development/repositories.rst [hunk 1]
```diff
@@ -2,37 +2,20 @@
 MOTECH Code Repositories
 ========================
 
-Each repository can be cloned from GitHub or Gerrit. If you're only interested in a copy of the code and will not be contributing, use the GitHub repo (no sign-in required). Otherwise use the Gerrit repo (sign-in required) where each commit will trigger a Jenkins build and be submitted for code review. Jenkins is our continuous integration (CI) system. Once your change is approved and merged by an authorized Gerrit reviewer, it will show up on the GitHub repo.
+We use GitHub to host our source code. Our core platform is available on the motech repository and our modules are hosted on the modules repository. Use the following comands to clone these repos.
```
Comment (frankhuster) [inline docs/source/development/repositories.rst]: Would it be easy to also make the `motech repository` and the `modules repository` hot links to the github web repo?

**Comment by frankhuster** [comment]
Small nitpick, otherwise LGTM

**Comment by craigappl** [comment]
@frankhuster  Updated the links. Sorry there were two commits, made a small typo in the first one.

**Comment by frankhuster** [comment]
LGTM

**Pull request merged**